filters/britishness/filter.py

```diff
@@ -0,0 +1,149 @@
+     'dangerous': 'dodgy',
+     'desirable': 'jammy',
+     'diaper': 'nappy',
+     'difficult': 'dodgy',
+     'disgusted': 'cheesed off',
+     'dish-towel': 'tea-towel',
+     'divided highway': 'dual carriageway',
+     'drug store': "chemist's",
+     'drunk': 'gassed',
+     'eavesdrop': 'earwig',
+     'employer': 'gaffer',
+     'engineer': 'engine driver',
+     'English muffin': 'crumpet',
+     'excellent': 'corking',
+     'exhausted': 'knackered',
+     'fed up': 'cheesed off',
+     'fool': 'prat',
+     'foolish': 'moony',
+     'foreman': 'gaffer',
+     'freeway': 'motorway',
+     'funds': 'lolly',
+     'gambler': 'punter',
+     'garbage': 'rubbish',
+     'garbage can': 'rubbish-bin',
+     'garbage collector': 'dustman',
+     'garbageman': 'dustman',
+     'gasoline': 'petrol',
+     'gear-shift': 'gear-lever',
+     'generator': 'dynamo',
+     'handcuffs': 'darbies',
+     'house': 'gaff',
+     'kitten': 'moggy',
+     'lavatory': 'loo',
+     'lies': 'porkies',
+     'liquor store': 'off-license',
+     'lucky': 'jammy',
+     'mail carrier': 'postman',
+     'mailbox': 'postbox',
+     'mailman': 'postman',
+     'man': 'bloke',
+     'math': 'maths',
+     'money': 'lolly',
+     'moron': 'pillock',
+     'mouth': 'gob',
+     'nonsense': 'codswallop',
+     'nose': 'hooter',
+     'nothing': 'nowt',
+     'odds and ends': 'odds and sods',
+     'offended': 'miffed',
+     'oil pan': 'sump',
+     'old man': 'geezer',
+     'optometrist': 'optician',
+     'outstanding': 'corking',
+     'patron': 'punter',
+     'pavement': 'road surface',
+     'pleasant': 'jammy',
+     'potato chips': 'potato crisps',
+     'railroad': 'railway',
+     'railway car': 'railway carriage',
+     'raincoat': 'mackintosh',
+     'schedule': 'timetable',
+     'sedan': 'saloon',
+     'shopper': 'punter',
+     'sneakers': 'gym shoes',
+     'spat': 'gobbed',
+     'spool of thread': 'reel of cotton',
+     'stroller': 'push-chair',
+     'subway': 'underground railway',
+     'sucker': 'punter',
+     'teeth': 'ivories',
+     'the brakes': 'the anchors',
+     'the movies': 'the cinema',
+     'the police': 'the fuzz',
+     'thumbtack': 'drawing pin',
+     'tired': 'knackered',
+     'trash': 'rubbish',
+     'trash can': 'rubbish-bin',
+     'trashcan': 'rubbish-bin',
+     'truck': 'lorry',
+     'turnpike': 'toll motorway',
+     'upset': 'miffed',
+     'vacation': 'holiday',
+     'vacuum cleaner': 'hoover',
+     'windshield': 'windscreen',
+     'wrench': 'spanner'
+    }
+
+    britwords += vocab_map.values()
+
+    britslang = ['tosser', 'blimey', 'wanker', 'chuffed', 'sod off', 'wonky', 'whinge', 'tenner', 'fiver', 'quid', 'toff', 'skive', 'scouse', 'scouser', 'cockney', 'nicked', 'nutter', 'gobsmacked', 'chap', 'bugger', 'anticlockwise', 'anti-clockwise', 'nosh', 'bollocks', 'ponce', 'bangers', 'telly', 'knickers', 'uni', 'albion', 'chunder', 'fortnight', 'grockel', 'kerfuffle', 'scrummy']
```

Comment:
Please split the list into multiple lines instead of having a big single line code.